demo/web/src/main/notebooks/A. Real-time table ops/A2 Filter and decorate .md

```diff
@@ -0,0 +1,69 @@
+# Filter and decorate
+
+Now that we have some data, let's work with it. In this notebook, we show how to decorate and filter our data.
+
+Let's start by creating some daily data. This could simulate something like daily stock prices or daily temperatures.
+
+```python
+start_time = convertDateTime("2020-01-01T00:00:00 NY")
+
+time_offset = Period("1D")
+daily_data = create_random_table(365, start_time, time_offset)
+```
+
+Now we decorate the data by adding its day of the week.<!--TODO what is a good way to describe "decorate"?-->
```

Comment:
I think decorate is an acceptable word here

Reply:
agree

Reply:
I'll remove my comment 🎉